src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -1001,6 +1015,23 @@ export default {
 				this.getTributeContainer().classList.remove(this.$style['tribute-container--focus-visible'])
 			}
 		},
+
+		showTributeManually(trigger) {
+			// Programmatic show of the tribute menu
+			this.$refs.contenteditable.focus()
+			let index = 0 // default to first collection '@'
+			if (trigger === ':') {
+				index += this.linkAutocomplete
+			} else if (trigger === '/') {
+				index += this.linkAutocomplete + this.emojiAutocomplete
+			}
```

Comment:
This works fine, but a bit hacky. If we change collections order, or init process, or make `@` optional, it's easy to lose this place where a sum of booleans is used to find the index.

Let's find it explicitly instead:
```suggestion
			const index = this.tribute.collection.findIndex((collection) => collection.trigger === trigger)
```